### PR TITLE
Implement functions to interact with the extension

### LINKED ISSUE #78: Implement function to interact with extensions.
## Feature Request

**Is your feature request related to a problem? Please describe.**  

A function of passing a value to the extension or receiving a value must be implemented.

**Describe the solution you'd like**  

**Describe alternatives you've considered**  

**Additional context**

### PR DESCRIPTION
**Pull request by HyeockJinKim**
# Summary 

- It closes #78 

## How to Review? 

- Please refer to my self-review submitted as the review.

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.0.8",
+  "version": "1.0.9",
```
Comment (HyeockJinKim) [inline lerna.json:5]: release new version to npm

File: packages/client/src/extension/extensionListener.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Listener, MessageType } from '@0auth/client';
+import { Supplier } from '@0auth/message';
+
+export function extensionListener(): Listener {
+  const functionMap: { [key: string]: Supplier<MessageType, unknown> } = {};
+  return {
+    add(key, func) {
+      functionMap[key] = func;
```
Comment (HyeockJinKim) [inline packages/client/src/extension/extensionListener.ts:8]: Store the functions to be executed in functionMap.

File: packages/client/src/extension/extensionListener.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    },
+    listen() {
+      window.addEventListener(
+        'message',
+        (event) => {
+          if (event.source !== window) return;
+
+          if (event.data.from && event.data.from === 'content') {
+            functionMap[event.data.type](event.data.message);
+          }
```
Comment (HyeockJinKim) [inline packages/client/src/extension/extensionListener.ts:19]: Execute different functions depending on the `event.data.type` to be executed.

File: packages/client/test/client.spec.ts [hunk 4]
```diff
@@ -244,3 +248,102 @@ describe('test hide property', () => {
...
+      }
+    },
+    postMessage: (message: { [key: string]: string }, targetOrigin: string) => {
+      if (targetOrigin !== undefined) {
+        const event = new MessageEvent('message', { data: message });
+        listenerQueue.forEach((listener) => listener(event));
+      }
+    },
+  };
+}
```
Comment (HyeockJinKim) [inline packages/client/test/client.spec.ts:283]: Declaration for mocking. Because node.js environment doesn't have `window` object.

**Review by HyeockJinKim — COMMENTED** [review]
self review

**Review by anthonyminyungi — APPROVED** [review]
LGTM.

**Pull request merged**